content/release-notes/2022-10-25.md

```diff
@@ -0,0 +1,16 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Add support of PostgreSQL 15.0 and its PostgreSQL extensions.
+For information about supported extensions, see [Available PostgreSQL extensions](https://neon.tech/docs/reference/compatibility/#available-postgresql-extensions).
+- Compute: Fix creation of databases with trailing spaces in database name.
+- Compute: Disable `wal_log_hints` by default.
```

Comment:
Why did we decide to disable wal_log_hints by default? Was it causing an issue?

Reply:
AFAIK, It was needed before for Neon to work correctly. Now it is not needed anymore and we change it back to PostgreSQL default.

Here is the PR https://github.com/neondatabase/neon/pull/2598

cc: @hlinnaka, @knizhnik

Reply:
Thank you. Suggested text below: